typescript/infra/src/agents/index.ts

```diff
@@ -145,28 +153,28 @@ export async function getAgentEnvVars(
     } else if (role === KEY_ROLE_ENUM.Validator) {
       envVars.push(
         `OPT_BASE_VALIDATOR_KEY=${strip0x(
-          gcpKeys[homeChainName + '-' + KEY_ROLE_ENUM.Validator].privateKey,
+          gcpKeys[outboxChainName + '-' + KEY_ROLE_ENUM.Validator].privateKey,
         )}`,
         `OPT_BASE_VALIDATOR_TYPE=hexKey`,
       );
       // Throw an error if the chain config did not specify the reorg period
-      if (valueDict.abacus.validator.reorg_period === undefined) {
+      if (valueDict.abacus.validator.configs[0].reorgPeriod === undefined) {
```

Comment:
Also under what circumstances is this undefined now? IIUC, this is now enforced at the compiler level right?